scripts/communityModules/VRBuild/VRBuildGrabscale.js

```diff
@@ -0,0 +1,499 @@
+    });
+    gizmoShapeIDs.push(gizmoZplusID);
+
+    gizmoZminusID = Entities.addEntity({
+        type: "Shape",        
+        shape: "Sphere",                       
+        name: "z-minus",
+        parentID: gizmoZaxisID,        
+        description: "",            
+        localPosition: { x: 0, y: -GIZMO_END_POSITION, z: 0},
+        lifetime: -1,
+        color: { r: 0, g: 0, b: 255 },
+        alpha: 1,
+        dimensions: { x: GIZMO_END_DIAMETER, y: GIZMO_END_DIAMETER, z: GIZMO_END_DIAMETER},
+        ignoreForCollisions: true,
+        userData: "{ \"grabbableKey\": { \"grabbable\": false, \"triggerable\": false } }"    
+    });
+    gizmoShapeIDs.push(gizmoZminusID);
+    ENTITIES_TO_UPDATE = [gizmoXplusID, gizmoXminusID, gizmoYplusID, gizmoYminusID, gizmoZplusID, gizmoZminusID];
+}
+
+function updateOverlay() {
+    overlayPosition = Vec3.sum(MyAvatar.position, Vec3.multiplyQbyV(MyAvatar.orientation, { x: 0, y: 2, z: -2 }));  
+    var text = [
+        "  GrabLeft: " + isGrabLeftInProgress + 
+        "  GrabRight: " + isGrabRightInProgress +
+        "  isGrabLeftAndRight: " + isGrabLeftAndRight,        
+        "  isScaling: " + isScaling + " newScale: " + newScale,
+        "  HandposLeft:  " + JSON.stringify(Vec3.subtract(handPositionLeft,MyAvatar.position)),
+        "  HandposRight:  " + JSON.stringify(Vec3.subtract(handPositionRight,MyAvatar.position)),        
+        "  dxplus: " + distanceToGizmoXplus.toFixed(DECIMAL_PRECISION) +
+        "  dyplus: " + distanceToGizmoYplus.toFixed(DECIMAL_PRECISION) +
+        "  dzplus: " + distanceToGizmoZplus.toFixed(DECIMAL_PRECISION),
+        "  dxmin: " + distanceToGizmoXmin.toFixed(DECIMAL_PRECISION) +
+        "  dymin: " + distanceToGizmoYmin.toFixed(DECIMAL_PRECISION) +
+        "  dzmin: " + distanceToGizmoZmin.toFixed(DECIMAL_PRECISION),        
+        "  isScaleModeX:  " + isScaleModeX + "  isScaleModeY:  " + isScaleModeY +"  isScaleModeZ:  " + isScaleModeZ,
+        "  index:   "+ index + "minimum:   " + minimum        
+    ].filter(Boolean).join('\n');
+
+    Overlays.editOverlay(overlayID, {
+        text: text,
+        position: overlayPosition      
+    });
+}
+
+function updateGizmoPositionRotation(gizmoPos, gizmoRot) {
+    var tempRotation = Quat.fromPitchYawRollRadians(0, 0, -Math.PI * 0.5 );
+    Entities.editEntity(gizmoCenterID, {
+        position: gizmoPos,        
+        rotation: Quat.multiply(gizmoRot, tempRotation)
+    });
+}
+
+function getJointIndex() {
+    RIGHT_JOINT = MyAvatar.getJointIndex("RightHandMiddle1");
+    LEFT_JOINT = MyAvatar.getJointIndex("LeftHandMiddle1");
+    if (RIGHT_JOINT === -1) {
+        RIGHT_JOINT = MyAvatar.getJointIndex("RightHand");        
+    }
+    if (LEFT_JOINT === -1) {
+        LEFT_JOINT = MyAvatar.getJointIndex("LeftHand");        
+    }
+    if (LEFT_JOINT === -1 || RIGHT_JOINT === -1) {
+        Window.alert("Avatar has no hand Joints aborting VRBuildGrabScale");
+        Script.stop();
```

Comment:
`Script.stop()` will unload the script, which means that it won't run the next time the user starts Interface.